src/scss/component-library.scss

```diff
@@ -0,0 +1,62 @@
+.component-library-wrapper {
+  background-color: white;
+  font-family: Arial, serif;
+  padding: 2rem;
+}
+
+.component-library__nav {
+  margin: 0 -5px;
+
+  a {
+    border-right: 1px solid gray;
+    color: black;
+    font-size: 12px;
+    padding: 3px 5px;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    &:hover {
+      color: var(--color-bus-dark-50);
```

Comment:
Did we agree to use this kind of variables or the regular ones?

Reply:
We did agree on SASS variables.